.gitignore

```diff
@@ -0,0 +1 @@
+dist/
```

Comment:
Add `build` script to your package.json that will build the application.
Add information to README.md about how this project should be running locally.